samples/csharp_dotnetcore/03.welcome-users/DeploymentScripts/MSbotClone/bot.recipe

```diff
@@ -4,7 +4,7 @@
     {
       "type": "endpoint",
       "id": "24",
```

Comment:
Are you generating this?  Kind of interesting your id - I think this is derived from config.bot file..

Reply:
I am running the script